### PR TITLE
signature_latte.txt is added to recipes folder and commited

### PR DESCRIPTION
**Pull request by prajwal2413**

### CODE REVIEW

File: recipes/signature_latte.txt
Comment (prajwal2413) [inline recipes/signature_latte.txt:1]: This is Joe's git cafe review

**Review by prajwal2413 — COMMENTED** [review]
code review on joe's git cafe

File: recipes/signature_latte.txt
Comment (prajwal2413) [reply recipes/signature_latte.txt:1]: yes

**Review by prajwal2413 — COMMENTED** [review]
modify the Menu file by adding the reference to the new signature_latte into the menu.

**Review by joegitcafe22 — CHANGES_REQUESTED** [review]
modify the Menu file by adding the reference to the new signature_latte into the menu.

**Review by joegitcafe22 — APPROVED** [review]
Approved changes

**Review by joegitcafe22 — CHANGES_REQUESTED** [review]
change menu

**Pull request merged**